monkey/monkey_island/cc/ui/src/components/types/MapNode.tsx

```diff
@@ -74,6 +74,8 @@ export default class MapNode {
     }
     else if (!this.island) { // No "clean" for island
       group_components.push('clean');
+    } else {
+      group_components.push('monkey');
```

Comment:
What did this fix?

Reply:
This fixed the island node when no agents were ran onto it